desktop/cmp/dash/canvas/DashCanvasViewModel.js

```diff
@@ -51,4 +61,40 @@ export class DashCanvasViewModel extends DashViewModel {
     setHeaderItems(items) {
         this.headerItems = items;
     }
+
+    //------------------------
+    // Implementation
+    //------------------------
+
+    onContentsResized({height}) {
+        if (!this.autoHeight) return;
+
+        const {id, containerModel} = this,
```

Comment:
Should we move this out into a public method on this model (`autosizeHeight()`?) so user's could do a one-off autosize of a view instead of just having it be a toggleable mode?

Reply:
I could see that being useful.  Would need to think a bit about the implementation (I think would require setting `flex: none` on the container in order to trigger the resize event, and then setting it back?). In the case of a one-off autoHeight event, we probably would want the new height persisted?